presence-sensing/tests/test_adc.cpp

```diff
@@ -0,0 +1,16 @@
+#include "catch2/catch.hpp"
+#include "presence-sensing/tests/mock_adc.hpp"
+#include "presence-sensing/core/presence_sensing_driver.hpp"
+
+SCENARIO("get readings called on ADC"){
```

Comment:
What does this test? It seems like it calls a mock instance directly and then checks the output of the mock, which just tests the mock and nothing else. That can be useful to make sure the mock is right but it doesn't really indicate anything about the shipping code.